src/project-tests/pomodoro-clock-tests.js

```diff
@@ -18,12 +18,12 @@ export default function createPomodoroClockTests() {
   }
 
   function getMinutes(str) {
-    const matches = (/^(\d{1,4})\s?([\.:,\/]\s?\d{2}.*)?$/g).exec(str);
+    const matches = (/(\d{1,2})\s?([\.:,\/]\s?\d{2}.*)?$/g).exec(str);
     return matches[1];
   }
 
   function getSeconds(str) {
-    const matches = (/^\d{1,4}\s?:\s?(\d{2})/g).exec(str);
+    const matches = (/\d{1,2}\s?[\.:,\/]\s?(\d{2})/g).exec(str);
```

Comment:
Same here about the documentation and examples.

Reply:
@tchaffee will do, I'll get on it in a day or two

Reply:
@tchaffee I managed to thoroughly test out the commented HTML. You can see a far fetched example and the regexes in my current code on: https://codepen.io/Roky/pen/jAKLXO. 
And by documentation, you ment putting it in the comments in the pomodoro test file?

Reply:
If we don't hear back I'll merge this. Unfortunately there is a merge conflict. After you pull --rebase upstream master, you'll be able to run yarn build and then squash commits / force push. Sorry for the trouble and thanks for contributing.

Reply:
@Rock-son I mean putting it right in the code, so your future self or someone else knows what the regex is doing. Something like this:

```
  // The regex will extract the seconds part of a time that looks like 10:07, where the 07 is
  // the number of seconds blah blah blah.
  function getSeconds(str) {
-    const matches = (/^\d{1,4}\s?:\s?(\d{2})/g).exec(str)
```

Just give a few examples of what will match and what won't match.